### PR TITLE
Update requests requirement as its old 

### PR DESCRIPTION
**Pull request by jeet4320**
## What
Update requests requirement as its old  and not compatible with latest libraries

### CODE REVIEW

**Review by santi-h — APPROVED** [review]

**Review by santi-h — CHANGES_REQUESTED** [review]
Can't merge without updating the version

**Comment by jeet4320** [comment]
done

File: setup.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
   download_url = 'https://github.com/bodyshopbidsdotcom/basegateway/tarball/{0}'.format(CURRENT_VERSION),
   keywords = ['api', 'gateway', 'http', 'REST'],
   install_requires = [
-    'requests==2.9.1'
+    'requests>=2.20.1'
```
Comment (santi-h) [inline setup.py]: actually why requiring that it should be `2.20.1` or newer? it's already been tested with `2.9.1` and `2.14.2`, so we can safely make it `>=2.9.1`

Comment (jeet4320) [reply setup.py]: because it is not compatible with awsebcli, docker, docker-compose

Comment (jeet4320) [reply setup.py]: ohh right

**Comment by jeet4320** [comment]
is it fine if we dont update version, getting below errors and I have to update all versions. 

```
ERROR: githubgateway 0.10 has requirement basegateway==0.12, but you'll have basegateway 0.13 which is incompatible.
ERROR: pywrike 0.5 has requirement basegateway==0.12, but you'll have basegateway 0.13 which is incompatible.
ERROR: pyrollbarapi 0.7 has requirement basegateway==0.12, but you'll have basegateway 0.13 which is incompatible.
```

**Comment by santi-h** [comment]
@jeet4320 you can try but I don't think pypi would let you change an already published version, every time you publish a package it has to be a new version. If pypi let you do that then apps would just randomly change without anyone knowing why.

**Comment by jeet4320** [comment]
@santi-h can you review below now

https://github.com/bodyshopbidsdotcom/githubgateway/pull/13
https://github.com/bodyshopbidsdotcom/PyWrike/pull/5
https://github.com/bodyshopbidsdotcom/PyRollbarAPI/pull/2

**Review by santi-h — APPROVED** [review]

**Pull request merged**